docs/application/dotnet/guides/app-management/cion.md

```diff
@@ -0,0 +1,434 @@
+# Cion
+
+The Cion API provides functionality to communicate with the other devices.
+
+The main features of the `Tizen.Applications.Cion` class include:
+
+- Communicating with other applications in server-client style.
+
+  You can communicate with other applications in [server-client](#server_client) style.
+
+- Communicating with other applications in group style.
+
+  You can communicate with other applications in [group](#group) style.
+
+And, you can generate communicate code using [TIDL](#tidl).
+
+## Prerequisites
+
+To enable your application to use the cion functionality, follow these steps:
+
+1.  To use the [Tizen.Applications.Cion](https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Cion.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
Thanks for the update to 'latest'.

However, it seems the link format changing to "/application/dotnet/..."  is not applied yet.

```suggestion
1.  To use the [Tizen.Applications.Cion](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Cion.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Reply:
it's changed